examples/p4est_2d_dgsem/elixir_navierstokes_NACA0012airfoil_mach08.jl

```diff
@@ -110,7 +110,7 @@ semi = SemidiscretizationHyperbolicParabolic(mesh, (equations, equations_parabol
 # ODE solvers
 
 # Run for a long time to reach a state where forces stabilize up to 3 digits
-tspan = (0.0, 1.0)
+tspan = (0.0, 10.0)
```

Comment:
What is the reason for this change?

Reply:
Originally this was 10, I don't now why this got reduced to 1, as we have the stabilization only at around that time.